### DISCUSSION TITLE
Is there a way to execute a single test?

### DISCUSSION THREAD

**Discussion by weinni2000** [Tecnativa/doodba-copier-template#504] [Q&A] [answered]
I know how to execute tests: 
https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md
Is there a way to execute a single test? Like test_model:TestModel.test_deprecated_geo_search__intersect_for_zip_1169 in the standard tests?

**Comment by pedrobaeza** [comment]
This is not specific of Doodba itself, but of Odoo. I think via `--test-tags` something can be done, but I haven't explored it.

**Comment by weinni2000** [comment, marked as answer]
Okay, I couldn't get it to run with the command from the documentation:
docker compose run --rm odoo addons init --test --with $modules

but with 
docker compose run --name doodba-odoo odoo  -d devel_test3 -u base_geoengine -i base_geoengine --test-enable --log-level=test --stop-after-init --test-tags /base_geoengine:TestModel.test_create_multipolygon_geojson_format -p 8088

it worked.

But I just realized that with the vscode tooling from doodba it's just runnning "Test and debug current module" (with the testfile selected) to have full breakpoint support in vscode. -> This should be mentioned in the documentation, because it's really awesome.
![Bildschirmfoto 2024-11-12 um 09 39 06](https://github.com/user-attachments/assets/8ac1e7f1-165e-44e7-bfe9-677f167b5482)
